app/main.py

```diff
@@ -1,25 +1,90 @@
-from abc import ABC
+from abc import ABC, abstractmethod
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __get__(self, instance: dict, owner: Any) -> int:
+        return instance.__dict__[self.name]
+
+    def __set__(self, instance: dict, value: int) -> None:
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError(f"Value must be "
+                             f"between {self.min_amount} "
+                             f"and {self.max_amount}")
+        instance.__dict__[self.name] = value
+
+    def __set_name__(self, owner: Any, name: Any) -> None:
```

Comment:
wrong annotation

Reply:
But I don't know what the annotation should be, there was no information in theory or videos